app/controllers/stories_controller.rb

```diff
@@ -358,22 +360,16 @@ def set_user_json_ld
         "@id": URL.user(@user)
       },
       "url": URL.user(@user),
-      "sameAs": [],
+      "sameAs": user_same_as,
       "image": ProfileImage.new(@user).get(width: 320),
       "name": @user.name,
-      "email": "",
-      "jobTitle": "",
+      "email": @user.email.presence,
+      "jobTitle": @user.employment_title.presence,
       "description": @user.summary.presence || "404 bio not found",
-      "disambiguatingDescription": [],
-      "worksFor": [
-        {
-          "@type": "Organization"
-        },
-      ],
-      "alumniOf": ""
-    }
-    set_user_profile_json_ld
-    set_user_same_as_json_ld
+      "disambiguatingDescription": user_disambiguating_description,
+      "worksFor": [user_works_for].compact,
+      "alumniOf": @user.education.presence
+    }.reject { |_, v| v.blank? }
```

Comment:
This is clever. Definitely missed this in my first pass-through of this PR :)

Reply:
A follow up thought: are there any keys that absolutely _must_ exist to render JSON-LD? If so, is there a chance that they'll be `blank?` and then not show up in the JSON-LD object?

Don't know if this is something that is likely to happen, but just want to double check.

Reply:
That's a good question! The keys that are absolutely required have hard-coded values (e.g. `"@context": "Schema.org"`) or values that should not evaluate to `true` when `.blank?` is called (e.g. `"@type": "Person"`). That being said, I'll double-check, but I think that I've mostly accounted for the edge cases. 😅